src/components/home/Home.jsx

```diff
@@ -0,0 +1,55 @@
+import { Navbar } from '../navbar/Navbar.jsx'
+import { Sidebar } from '../sidebar/Sidebar.jsx'
+import { MainContent } from '../main/MainContent.jsx'
```

Comment:
Try to add barrel imports

Reply:
Sure, will do it once start building separate pages